Indicators/PercentagePriceOscillator.cs

```diff
@@ -54,7 +54,7 @@ protected override decimal ComputeNextValue(IndicatorDataPoint input)
         {
             var value = base.ComputeNextValue(input);
 
-            return (IsReady && Slow != 0) ? 100 * value / Slow : 0m;
+            return Slow != 0 ? 100 * value / Slow : 0m;
```

Comment:
The numerator is just the difference of two moving averages (can be positive or negative, small or large). In theory, this oscillator is unbounded, but it oscillates around the value of zero. I think the case of a zero denominator should be zero (as done in the TA-lib source code).